tests/test_cmdline.py

```diff
@@ -557,7 +704,11 @@ def test_check_json__simple_project__can_report_both_undeclared_and_unused(
         "version": version(),
     }
     output, returncode = run_fawltydeps_function(
-        "--check", "--json", f"--code={tmp_path}", f"--deps={tmp_path}"
+        "--check",
+        "--json",
+        f"--code={tmp_path}",
+        f"--deps={tmp_path}",
+        f"--pyenv={tmp_path}",
```

Comment:
Why add `--pyenv` here?

Reply:
I want to actually verify that what a `PyEnvSource` looks like in the expected output (lines 667-670).

Without adding `--pyenv`, I would have to rely on this Python environment to be auto-discovered, which only happens if `tmp_path` is passed as `basepath`, or if we had instrumented `run_fawltydeps_function()` to run with `tmp_path` as the current directory.